docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -6,7 +6,7 @@ Google Sheets connector
 
   <img src="../_static/img/google-sheets.png" class="connector-logo">
 
-The Google Sheets connector allows reading `Google Sheets <https://www.google.com/sheets/about/>`_ spreadsheets as tables in Trino.
+The Google Sheets connector allows reading and writing `Google Sheets <https://www.google.com/sheets/about/>`_ spreadsheets as tables in Trino.
```

Comment:
@mosabua Do you want to review this docs change?

Reply:
@mosabua ping

Reply:
Thankfully he has already reviewed the docs